.eslintrc.js

```diff
@@ -8,9 +8,15 @@ module.exports = {
   },
 
   "env": {
+    "browser": true,
     "es6": true,
+    "mocha": true,
     "node": true,
-    "browser": true
+  },
+
+  "globals": {
+    "expect": true,
```

Comment:
If I remove this I get `warning  'sinon' is not defined` and `warning  'expect' is not defined` when running `npm run lint`
```